### PR TITLE
Add 7.0.0 RC-1 and 6.3.0 release notes

### PR DESCRIPTION
**Pull request by schmittjoseph**
Add rolling releases notes for 7.0.0 RC-1, along with updating our release notes templates to include experimental features.

This is a work-in-progress, feel free to comment with any additions.

### CODE REVIEW

File: documentation/releaseNotes/releaseNotes.v7.0.0-rc.1.rolling.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+Today we are releasing the next official preview of the `dotnet monitor` tool. This release includes:
+
+- With this release, support for experimental features is added. Users can now enable an experimental feature through configuration flags when launching `dotnet monitor` to enable features that are still under development and not yet fully production-ready. (#2506)
+- 🔬 Add collect stacks api and action (#2512, #2525)
+- Avoid Kestrel address override warning (#2535)
+- Only create default diagnostic port when running in listen mode (#2523)
```
Comment (jander-msft) [inline documentation/releaseNotes/releaseNotes.v7.0.0-rc.1.rolling.md:6]:
```suggestion
- Create default diagnostic port under default shared path when running in listen mode (#2471, #2523)
```

File: documentation/releaseNotes/releaseNotes.v7.0.0-rc.1.rolling.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
+Today we are releasing the next official preview of the `dotnet monitor` tool. This release includes:
```
Comment (jander-msft) [inline documentation/releaseNotes/releaseNotes.v7.0.0-rc.1.rolling.md:1]:
```suggestion
Today we are releasing the official 7.0 Release Candidate of the `dotnet monitor` tool. This release includes:
```

File: documentation/releaseNotes/releaseNotes.md [hunk 3]
```diff
@@ -1,6 +1,8 @@
 Today we are releasing the next official preview of the `dotnet monitor` tool. This release includes:
 
-- ⚠️ [Here is a breaking change we did and its work item] (#737)
-- [Here is a new feature we added and its work item] (#737)
+- ⚠️ [Here is a breaking change we did and it's work item] (#737)
```
Comment (kkeirstead) [inline documentation/releaseNotes/releaseNotes.md:3]:
```suggestion
- ⚠️ [Here is a breaking change we did and its work item] (#737)
```

File: documentation/releaseNotes/releaseNotes.md [hunk 4]
```diff
@@ -1,6 +1,8 @@
...
+- 🔬 [Here is an experimental feature we did and it's work item] (#737)
```
Comment (kkeirstead) [inline documentation/releaseNotes/releaseNotes.md:4]:
```suggestion
- 🔬 [Here is an experimental feature we did and its work item] (#737)
```

File: documentation/releaseNotes/releaseNotes.md [hunk 5]
```diff
@@ -1,6 +1,8 @@
...
+- [Here is a new feature we added and it's work item] (#737)
```
Comment (kkeirstead) [inline documentation/releaseNotes/releaseNotes.md:5]:
```suggestion
- [Here is a new feature we added and its work item] (#737)
```

File: documentation/releaseNotes/releaseNotes.template.md [hunk 6]
```diff
@@ -1,6 +1,8 @@
 Today we are releasing the next official preview of the `dotnet monitor` tool. This release includes:
 
 - ⚠️ [Here is a breaking change we did and it's work item] (#737)
+- 🔬 [Here is an experimental feature we did and it's work item] (#737)
```
Comment (kkeirstead) [inline documentation/releaseNotes/releaseNotes.template.md:4]:
```suggestion
- 🔬 [Here is an experimental feature we did and its work item] (#737)
```

File: documentation/releaseNotes/releaseNotes.template.md [hunk 7]
```diff
@@ -1,6 +1,8 @@
...
 - [Here is a new feature we added and it's work item] (#737)
```
Comment (kkeirstead) [inline documentation/releaseNotes/releaseNotes.template.md:5]:
```suggestion
- [Here is a new feature we added and its work item] (#737)
```

File: documentation/releaseNotes/releaseNotes.template.md [hunk 8]
```diff
@@ -1,6 +1,8 @@
 Today we are releasing the next official preview of the `dotnet monitor` tool. This release includes:
 
 - ⚠️ [Here is a breaking change we did and it's work item] (#737)
```
Comment (kkeirstead) [inline documentation/releaseNotes/releaseNotes.template.md:3]:
```suggestion
- ⚠️ [Here is a breaking change we did and its work item] (#737)
```

File: documentation/releaseNotes/releaseNotes.v7.0.0-rc.1.rolling.md [hunk 9]
```diff
@@ -0,0 +1,10 @@
+Today we are releasing the official 7.0 Release Candidate of the `dotnet monitor` tool. This release includes:
```
Comment (jander-msft) [inline documentation/releaseNotes/releaseNotes.v7.0.0-rc.1.rolling.md:1]: Please rename this file to use the candidate version number: 7.0.0-rc.1.22479.8

File: documentation/releaseNotes/releaseNotes.v6.3.0.md [hunk 10]
```diff
@@ -0,0 +1,9 @@
+Today we are releasing the 6.3.0 build of the `dotnet monitor` tool. This release includes:
+
+- Add metadata for Azure egress (#2290)
+- Support restrictive SAS tokens for Azure egress (#2322, #2366)
+- Avoid Kestrel address override warning (#2535)
+- Check read permissions for JSON configuration files (#2232)
+- Improve `operations` Api to show RuntimeInstanceId and support filtering (#2136, #2312)
```
Comment (jander-msft) [inline documentation/releaseNotes/releaseNotes.v6.3.0.md:7]:
```suggestion
- Improve `operations` Api to show RuntimeInstanceId and support filtering (#2265, #2360)
```

File: documentation/releaseNotes/releaseNotes.v6.3.0.md [hunk 11]
```diff
@@ -0,0 +1,9 @@
+Today we are releasing the 6.3.0 build of the `dotnet monitor` tool. This release includes:
+
+- Add metadata for Azure egress (#2290)
+- Support restrictive SAS tokens for Azure egress (#2322, #2366)
+- Avoid Kestrel address override warning (#2535)
+- Check read permissions for JSON configuration files (#2232)
```
Comment (jander-msft) [inline documentation/releaseNotes/releaseNotes.v6.3.0.md:6]:
```suggestion
- Check read permissions for JSON configuration files (#2272)
```

File: documentation/releaseNotes/releaseNotes.v6.3.0.md [hunk 12]
```diff
@@ -0,0 +1,9 @@
+Today we are releasing the 6.3.0 build of the `dotnet monitor` tool. This release includes:
+
+- Add metadata for Azure egress (#2290)
+- Support restrictive SAS tokens for Azure egress (#2322, #2366)
```
Comment (jander-msft) [inline documentation/releaseNotes/releaseNotes.v6.3.0.md:4]:
```suggestion
- Support restrictive SAS tokens for Azure egress (#2325, #2394)
```

File: documentation/releaseNotes/releaseNotes.v6.3.0.md [hunk 13]
```diff
@@ -0,0 +1,9 @@
+Today we are releasing the 6.3.0 build of the `dotnet monitor` tool. This release includes:
+
+- Add metadata for Azure egress (#2290)
```
Comment (jander-msft) [inline documentation/releaseNotes/releaseNotes.v6.3.0.md:3]:
```suggestion
- Add metadata for Azure egress (#2371)
```

File: documentation/releaseNotes/releaseNotes.v6.3.0.md [hunk 14]
```diff
@@ -0,0 +1,9 @@
...
+- Support restrictive SAS tokens for Azure egress (#2322, #2366)
+- Avoid Kestrel address override warning (#2535)
+- Check read permissions for JSON configuration files (#2232)
+- Improve `operations` Api to show RuntimeInstanceId and support filtering (#2136, #2312)
+- Fixed an issue (#2526) where the `config` command could throw an exception (#2530)
```
Comment (jander-msft) [inline documentation/releaseNotes/releaseNotes.v6.3.0.md:8]:
```suggestion
- Fixed an issue (#2526) where the `config` command could throw an exception (#2551)
```

File: documentation/releaseNotes/releaseNotes.v6.3.0.md [hunk 15]
```diff
@@ -0,0 +1,9 @@
...
+- Fixed an issue where running `config show --show-sources` could incorrectly report `ChainedConfigurationProvider` as the source (#2550)
```
Comment (jander-msft) [inline documentation/releaseNotes/releaseNotes.v6.3.0.md:9]:
```suggestion
- Fixed an issue where running `config show --show-sources` could incorrectly report `ChainedConfigurationProvider` as the source (#2553)
```

File: documentation/releaseNotes/releaseNotes.v6.3.0.md [hunk 16]
```diff
@@ -0,0 +1,9 @@
+Today we are releasing the 6.3.0 build of the `dotnet monitor` tool. This release includes:
+
+- Add metadata for Azure egress (#2290)
+- Support restrictive SAS tokens for Azure egress (#2322, #2366)
+- Avoid Kestrel address override warning (#2535)
```
Comment (jander-msft) [inline documentation/releaseNotes/releaseNotes.v6.3.0.md:5]:
```suggestion
- Avoid Kestrel address override warning (#2576)
```

**Review by jander-msft — APPROVED** [review]

**Pull request merged**